.gitignore

```diff
@@ -0,0 +1,2 @@
+/.idea/
```

Comment:
can you specify this for yourself locally and remove this file? This is not generally related to this repo and should be handled by users themselves, if they choose to use an IDE which generates this stuff.